.nuspec/Xamarin.Forms.MultiTargeting.targets

```diff
@@ -40,16 +40,36 @@
     <None Include="**\Win32\*.cs" />
   </ItemGroup>
 
+
+  <!-- Android -->
+  <PropertyGroup Condition="$(TargetFramework.StartsWith('net6.0-android'))">
+    <DefineConstants>$(DefineConstants);MONOANDROID</DefineConstants>
```

Comment:
I think you could use the ANDROID define instead, it is there by default in .NET 6.